doc/classes/PhysicsShapeQueryParameters3D.xml

```diff
@@ -27,6 +27,7 @@
 		</member>
 		<member name="motion" type="Vector3" setter="set_motion" getter="get_motion" default="Vector3(0, 0, 0)">
 			The motion of the shape being queried for.
+			[b]Note:[/b] Many methods ignore this property.
```

Comment:
That leaves the reader a bit hanging - how to know which methods ignore this property or not? Assuming this is properly documented in `PhysicsDirectSpaceState3D` method descriptions, I'd mention that here.
```suggestion
			[b]Note:[/b] Only some methods of [PhysicsDirectSpaceState3D] that take this class as parameter make use of this property. See their respective descriptions for details.
```